draft-ietf-ice-rfc5245bis.xml

```diff
@@ -782,56 +767,56 @@ it to send a check.
 consequence of the receipt of a connectivity check from the peer.
 </t>
 
-<t hangText="VALID LIST:"> An ordered set of candidate pairs for a
+<t hangText="Valid List:"> An ordered set of candidate pairs for a
 media stream that have been validated by a successful STUN
 transaction.
 </t>
 
-<t hangText="Check List Set:"> An ordered list of CHECK LISTs.
+<t hangText="Check List Set:"> The ordered list of all check lists.
+The order is determined by each ICE usage.
 </t>
 
-<t hangText="Full:"> An ICE implementation that performs the complete
-set of functionality defined by this specification.
+
+<t hangText="Full Implementation:"> An ICE implementation that
+performs the complete set of functionality defined by this
+specification.
 </t>
 
-<t hangText="Lite:"> An ICE implementation that omits certain
+<t hangText="Lite Implementation:"> An ICE implementation that omits certain
 functions, implementing only as much as is necessary for a peer
 implementation that is full to gain the benefits of ICE. Lite
 implementations do not maintain any of the state machines and do not
 generate connectivity checks.
 </t>
 
-<t hangText="Controlling Agent:"> The ICE agent that is responsible
-for selecting the final choice of candidate pairs and signaling them
-through STUN. In any session, one agent is always controlling. The
+<t hangText="Controlling Agent:"> The ICE agent that nominates a
+candidate pair.  In any session, one agent is always controlling. The
 other is the controlled agent.
 </t>
 
-<t hangText="Controlled Agent:"> An ICE agent that waits for the
-controlling agent to select the final choice of candidate pairs.
+<t hangText="Controlled Agent:"> The ICE agent that waits for the 
+controlling agent to nominate a candidate pair.
 </t>
 
```

Comment:
Should we say that the controlled agent is responsible for selecting a candidate nominated by the controlling agent?